src/main/scala/view/scalaFX/FXControllers/BaseAppController.scala

```diff
@@ -0,0 +1,68 @@
+package view.scalaFX.FXControllers
+
+import model.Bunny
+import scalafx.animation.Timeline
+import scalafx.application.Platform
+import scalafx.collections.ObservableBuffer
+import scalafx.scene.image.Image
+import scalafx.scene.layout.{AnchorPane, Background, BackgroundImage, BackgroundPosition, BackgroundRepeat, BackgroundSize}
+import scalafx.util.Duration
+import scalafxml.core.macros.sfxml
+import view.scalaFX.components.BunnyView
+
+trait BaseAppControllerInterface {
+  def initialize(bunnies: Seq[Bunny]): Unit
+}
+
+@sfxml
+class BaseAppController(private val simulationPane: AnchorPane,
+                        private val graphPane: AnchorPane,
+                        private val mutationChoicePane: AnchorPane,
+                        private val factorChoicePane: AnchorPane,
+                        private val graphChoicePane: AnchorPane,
+                        private var bunnyTimeline: Timeline) extends BaseAppControllerInterface {
+
+
+  private var bunnyViews: Seq[BunnyView] = Seq.empty
+  private var bunnyTimelines: Seq[Timeline] = Seq.empty
+
+  def initialize(bunnies: Seq[Bunny]): Unit = {
+    if (bunnyTimelines.nonEmpty) bunnyTimelines.foreach(bunny => bunny.stop())
```

Comment:
Anche qui in teoria si può usare "_", cioè `if (bunnyTimelines.nonEmpty) bunnyTimelines.foreach(_.stop())`